### DISCUSSION TITLE
Export Song Info to txt file

### DISCUSSION THREAD

**Discussion by JaL-P** [ciderapp/Cider#707] [Feature Request]
Adding a way to export song and artist information to a text file with an option window to adjust what can be displayed (for example $s for Song name and $a for Artist) 

Alternatively, a way to integrate with another project like [Snip](https://github.com/dlrudie/Snip) or [Tuna](https://github.com/univrsal/tuna)

**Comment by quacksire** [comment]
Plugin system is implemented. If anyone wants to make a plugin to close this, feel free

**Comment by ChaseIngebritson** [comment]
@JaL-P I've thrown together a plugin that should accomplish what you wanted. Currently it's in my fork, but I'll create a separate repo to host it, format it to be bundled as a third party plugin, and link to it here this week.

**Comment by ChaseIngebritson** [comment]
So I finally got around to creating that repo. It can be found here: https://github.com/ChaseIngebritson/text-output-plugin with installation instructions found in the README.

It does not currently run on the Windows Store version, so I'm going to have to do a bit of debugging. However, I'll try and keep updating it as Cider continues to update.
